docs/api-reference.md

```diff
@@ -203,7 +203,18 @@ init(initialProps) -> void
 
 `init` is called exactly once when a new instance of a component is created. It can be used to set up the initial `state`, as well as any non-`render` related values directly on the component.
 
-`init` is the only place where you can assign to `state` directly, as opposed to using `setState`:
+Use `setState` inside of `init` to set up your initial component state:
+
+```lua
+function MyComponent:init()
+	self:setState({
+		position = 0,
+		velocity = 10
+	})
+end
+```
+
+In older versions of Roact, `setState` was disallowed in `init`, and you would instead assign to `state` directly. This is still acceptable, but it's simpler to use `setState`:
```

Comment:
Wording right before the colon here is a bit confusing as it sounds like you're about to demonstrate use of `setState`. Could you construct the sentence differently? Maybe "It's simpler to use `setState`, but it's still acceptable to assign it directly:"